docs/HowToAddAnOperation.md

```diff
@@ -0,0 +1,62 @@
+# How to support to process a new ONNX Operation
+
+The text below describe the list of actions needed to successfully process a new ONNX operation. We are adding here support for an existing operation, thus when parsing the ONNX list of operation form the ONNX repository, this operation is already present. The step below explain how to add the necessary support to process that operation and lower it to code that can be executed using the LLVM compiler backend.
+
+In the example below, we assume that we add support for the Concat ONNX operation. All paths are relatives to the root directory of onnx-mlir distribution.
+
+
+## Generate the proper ONNX.td.inc
+
+The first step is to add support so that MLIR can determine the output type and shape from its input variables and parameters. This step is called “Shape Inference.” The first step is to indicate in the automatically generated `ONNXOps.td.inc` that the new operation needs to invoke shape inference.
```

Comment:
needs to invoke shape inference -> implements the shape inference interface method?

Reply:
yes, tx